Replicator/c4Replicator.cc

```diff
@@ -89,43 +95,65 @@ bool c4address_fromURL(C4String url, C4Address *address, C4String *dbName) C4API
     slice str = url;
 
     auto colon = str.findByteOrEnd(':');
-    if (!colon)
+    if (!colon) {
+        // Can't log URL yet unless we add a function to strip out a potential password
+        WarnError("c4address_fromURL: Invalid URL (missing colon)");
```

Comment:
This is overkill, logging every possible way a URL could be invalid. We could just have one place where we log the URL and say it's invalid. (Don't we already do that in the error we return from the replicator?)